dev-packages/cli/README.md

```diff
@@ -318,6 +318,15 @@ The property `theiaPlugins` describes the list of plugins to download, for examp
 }
 ```
 
+The property `theiaPluginsExcludeIds` can be used to declare the list of plugin `ids` to exclude when using extension-packs.
+The `ids` referenced by the property will not be downloaded when resolving extension-packs, and can be used to omit extensions which are problematic or unwanted. The format of the property is as follows:
+
+```json
+"theiaPluginsExcludeIds": [
```

Comment:
Maybe we can simplify and use "theiaPluginsExcluded"? It seems that excluding by ID is the logical, intuitive way to go and you documented it too.